OpenShift/Standard/serviceaccelerator.md

```diff
@@ -0,0 +1,560 @@
+
+Please see the associated links for RHCOS Configuration, Hardening, Compliance Scanning and Installation.
+<td><ol type="1">
+<li><p>Creating Red Hat Enterprise Linux CoreOS (RHCOS) machines: <a href="https://docs.openshift.com/container-platform/4.6/installing/installing_bare_metal/installing-bare-metal.html#creating-machines-bare-metal">https://docs.openshift.com/container-platform/4.6/installing/installing_bare_metal/installing-bare-metal.html#creating-machines-bare-metal</a></p></li>
+<li><p>RHEL CoreOS Compliance Scanning in OpenShift 4: <a href="https://www.openshift.com/blog/rhel-coreos-compliance-scanning-in-openshift-4">https://www.openshift.com/blog/rhel-coreos-compliance-scanning-in-openshift-4</a></p></li>
+<li><p>Hardening RHCOS: <a href="https://docs.openshift.com/container-platform/4.6/security/container_security/security-hardening.html">https://docs.openshift.com/container-platform/4.6/security/container_security/security-hardening.html</a></p></li>
+</ol></td>
+</tr>
+<td><strong>Certificate and Key Management</strong></td>
+<td></td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>How does the service you are evaluating handle Certificate and Key Management?</td>
+<td>All certificates for internal traffic is managed by OpenShift and rotated automatically. Egress (Proxy) traffic CA is configurable. Ingres traffic is configurable
+<td><ol type="1">
+<li><p>User-provided certificates for the API server: <a href="https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/user-provided-certificates-for-api-server.html">https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/user-provided-certificates-for-api-server.html</a></p></li>
+</ol></td>
+</tr>
+<td><strong>Encryption</strong></td>
+<td></td>
+<td></td>
+</tr>
+<tr class="odd">
+<td>How does the service you are evaluating handle Encryption at Rest?</td>
+<td>Encrypting etcd data - To be updated
+<td><ol type="1">
+<li><p>Encrypting ETCD: <a href="https://docs.openshift.com/container-platform/4.6/security/encrypting-etcd.html">https://docs.openshift.com/container-platform/4.6/security/encrypting-etcd.html</a></p></li>
+</ol></td>
+</tr>
+<tr class="odd">
+<td>How does the service you are evaluating handle Encryption in Transit?</td>
+<td>OpenShift 4.7 adds IPSec support for the OVN-Kubernetes CNI - To be updated
+<td><ol type="1">
+<li>To Be Added</li>
+</ol></td>
+</tr>
+<td><strong>Network Policy and Security</strong></td>
+<td></td>
+<td></td>
+</tr>
+
+</tr>
+<!--
```

Comment:
i imagine this is intended as a "preview"? Or is it accidentally left in?

It'd be nicest to have a comment to remove this before the PR is merged instead of leaving it as a comment, since this interferes with the markdown preview side of things, making the doc quite hard to review.

Reply:
Sorry for the late reply, I actually converted the format to Asciidoc and submitted subsequent pull requests.